doc/src/pair_sw.rst

```diff
@@ -157,16 +168,52 @@ used for anything and can be set to 0.0 if desired.
 This is also true for the parameters in :math:`\phi_3` that are
 taken from the ij and ik pairs (:math:`\sigma`, *a*, :math:`\gamma`)
 
+The *modify* keyword computes the energy E of a system of atoms, whose
+formula is the same as the Stillinger-Weber potential. The only modification
+is in the three-body term, where :math:`\delta = \cos \theta_{ijk} - \cos \theta_{0ijk}`
+is modified with the following function:
+
+.. math::
+
+  g_C(\delta) & = \left\{ \begin{array} {r@{\quad:\quad}l}
+    1 & \delta < \delta_1 \\
+    \frac{1}{2} + \frac{1}{2} \cos \left( \pi \frac{\delta-\delta_1}{\delta_2 - \delta_1} \right) &
+      \delta_1 < \delta < \delta_2 \\
+    0 & \delta > \delta_2
+    \end{array} \right. \\
+
+
+The *modify* keyword is designed for simulations of materials when
+distinguishing three-body angles are necessary, such as borophene
+and transition metal dichalcogenide, which cannot be described 
+by the original code for the Stillinger-Weber potential. Validation, 
+benchmark tests, and applications of the *modify* keyword can be found in
+:ref:`(Jiang_1) <Jiang1>` and :ref:`(Jiang_2) <Jiang2>`.
+
 ----------
 
-.. include:: accel_styles.rst
+Styles with a *gpu*, *intel*, *kk*, *omp*, or *opt* suffix are
```

Comment:
This change reverts changes in upstream. Please undo.

Reply:
Removed these changes and add a new file "pair_sw_mod.rst".

Reply:
Sorry, I don't see this properly restored. Perhaps you missed committing all changes or pushing all of them.
Besides, I would prefer to have just one .rst file for pair styles sw and sw/mod. Pair style tersoff/mod is not a good template to follow here. Overall, we are trying to balance manual pages to avoid having too many with lots of redundant content, but also split those with too many variants in the same file, so that we reach a balance between better maintainability (i.e. fewer files) and better readability (i.e. less convoluted pages)

Reply:
Thanks for the check, I indeed forgot to commit all changes. This issue is now fixed and the rst files are merged into a single one.

Reply:
There were a few more changes missing. I have updated the PR to properly include and link the docs, reordered the paragraphs and fixed a bunch more things.